Automattic/wp-calypso
# Add Jetpack AI Yearly supporting code

Related to D117295-code

## Proposed Changes

We're adding Jetpack AI Yearly offering. We need to adjust some code in wp-calypso to support the new plan as Jetpack product alongside Jetpack AI Monthly.

- Remove MONTHLY_ONLY code logic
- Add jetpack_ai_yearly to product constants
- Add jetpack_ai_yearly to product list, copy, icons

## Testing Instructions

* Ensure you have D117295-code patched on your sandbox and point `public-api.wordpress.com` to it (with store sandbox enabled)
* Checkout the diff and run `yarn start-jetpack-cloud-p` and `yarn start`
* Go to `http://jetpack.cloud.localhost:3001/pricing`
* Jetpack AI on pricing should display yearly plan (instead of monthly):

<img width="631" alt="CleanShot 2023-07-28 at 10 12 35@2x" src="https://github.com/Automattic/wp-calypso/assets/8419292/82804b2f-330d-4910-a800-e109681059fe">

* The "Get" CTA button should lead to `https://wordpress.com/checkout/jetpack/jetpack_ai_yearly`
* Now, switch url to `https://calypso.localhost:3000/checkout/jetpack/jetpack_ai_yearly`
* See that copy is updated (text "Plan Subscription" vs "Billed yearly") and dropdown with monthly/yearly option is displayed:
<img width="679" alt="CleanShot 2023-07-28 at 10 17 48@2x" src="https://github.com/Automattic/wp-calypso/assets/8419292/d00908e3-acea-4d8c-a9b2-00cb149672aa">


## Pre-merge Checklist
